src/Domain/LeanCode.DomainModels.MassTransitRelay/Outbox/PeriodicEventsPublisher.cs

```diff
@@ -40,7 +40,7 @@ public async Task ExecuteAsync(CancellationToken stoppingToken)
 
             var events = await FetchUnpublishedEventsAsync();
 
-            logger.Information("There are {EventsCount} to be published", events.Count);
+            logger.Debug("There are {EventsCount} to be published", events.Count);
```

Comment:
Yeah, that was the log that I missed - There are X **events** to be published.

Reply:
Fixed